packages/nx/src/command-line/init/implementation/angular/standalone-workspace.ts

```diff
@@ -120,10 +121,13 @@ function createNxJson(
     if (fileExists(join(repoRoot, 'eslint.config.js'))) {
       inputs.push('{workspaceRoot}/eslint.config.js');
     }
-    nxJson.targetDefaults.lint = { inputs };
+    nxJson.targetDefaults.lint = { inputs, cache: true };
```

Comment:
This shouldn't overwrite what the `createNxJsonFile` helper (updated in this PR) already sets based on the cacheable operations.